subsys/logging/log_backend_spinel.c

```diff
@@ -87,8 +90,28 @@ static int write(uint8_t *data, size_t length, void *ctx)
 		platformUartPanic();
 		otPlatUartSend(data, length);
 	} else {
-		otPlatLog(CONFIG_LOG_BACKEND_SPINEL_LEVEL,
-			  OT_LOG_REGION_PLATFORM, "%s", data);
+		switch (last_log_level) {
+		case LOG_LEVEL_ERR:
+			otPlatLog(OT_LOG_LEVEL_CRIT,
```

Comment:
"codesizewise" it might be better to set local variable to target level and call`otPlatLog` after switch/case.

Reply:
Sure, good idea, done.